egs/wsj/s5/utils/lattice_depth_per_frame.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright 2015  Brno University of Technology (Author: Karel Vesely)
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+# Extract lattice-depth for each frame.
```

Comment:
this script lattice_depth_per_frame.sh should probably be in steps/conf/ unless and until we need it for other purposes.  It's not much more than a wrapper for the C++ program-- and in fact for something as simple as this, unless you use it a lot I'd tend to just include the lines of script directly rather than invoking a separate script.